theano/tensor/basic.py

```diff
@@ -3963,16 +3963,53 @@ def shape_padright(t, n_ones=1):
 
 
 @constructor
+def shape_padaxis(t, axis):
+    """Reshape `t` by adding 1 at the dimension `axis`.
+
+    See also: `shape_padleft`, `shape_padright` and `Dimshuffle`
+    """
+    _t = as_tensor_variable(t)
+
+    pattern = [i for i in xrange(_t.type.ndim)]
+    pattern.insert(axis, 'x')
+    return DimShuffle(_t.broadcastable, pattern)(_t)
+
+
+@constructor
 def stack(*tensors):
     """Insert the arguments as slices into a tensor of 1 rank greater.
 
     The size in dimension 0 of the result will be equal to the number
     of tensors passed.
 
+    Note: The interface stack(*tensors) is deprecated, you should use
+    stack(tensors, axis=0) insted.
+
+    :Parameters:
```

Comment:
Parameters docstring format is somehow inconsistant in Theano. @lamblin Should we use 
```
"""
:Parameters: 
- `parameter_name` : type
    description
"""
```

or this

```
"""
Parameters
----------
parameter_name : type
    description
"""
```